models/organization/org.go

```diff
@@ -195,16 +196,22 @@ func (org *Organization) CanCreateRepo() bool {
 // FindOrgMembersOpts represensts find org members conditions
 type FindOrgMembersOpts struct {
 	db.ListOptions
-	OrgID      int64
-	PublicOnly bool
+	Doer     *user_model.User
+	IsMember bool
```

Comment:
What does `IsMember` mean?

Reply:
Is the Doer member of the Org we query info from

Reply:
@lunny should i rename to `IsDoerMember` ?

Reply:
Looks better.

Reply:
done